README.md

```diff
@@ -67,13 +78,14 @@ $item->addPrice(13.37);
 $xmlOutput = $exporter->serializeItems([$item], 0, 1, 1);
 ```
 
+You may want to include the header `header('Content-type: text/xml')`.
```

Comment:
As this depends on the type of system where the library is integrated, the information is not needed. Furthermore I don't think that anything is breaking if the `Content-Type` is different than `text/xml`. Also in the documentation nothing is mentioned about that.

Reply:
Can confirm that the `Content-Type` doesn't matter for consuming the export file. It's just a nice thing to have if you want to view the export file in a browser.

I concur with @mmachatschek that this also is highly specific to the system generating the export, as in many cases when using frameworks, the headers might already have been sent once the body is generated, so using `header()` would in fact result in an error.